app/views/layouts/application.html.erb

```diff
@@ -96,6 +96,9 @@
               <li class="govuk-footer__inline-list-item">
                 <%= link_to t('layout.accessibility'), accessibility_path, class: 'govuk-footer__link' %>
               </li>
+              <li class="govuk-footer__inline-list-item">
+                <%= link_to t('layout.terms_conditions'), terms_conditions_path, class: 'govuk-footer__link' %>
```

Comment:
I *think* these T&Cs only apply to the candidate facing bit, the providers apparently have different terms: https://trello.com/c/4zjNLeCu/1148-tcs

Reply:
Ah! I didn’t realise providers where using the same template (or app, in fact). Should provider and candidate views use a different template (or something like that)?